packages/lang-server/src/usage.ts

```diff
@@ -80,19 +80,28 @@ export const getSearchElementFullName = (context: PositionContext, token: string
   return undefined
 }
 
-export const getReferencingFiles = async (workspace: EditorWorkspace, fullName: string): Promise<string[]> => {
-  try {
-    const id = ElemID.fromFullName(fullName)
-    return await workspace.getElementReferencedFiles(id)
-  } catch (e) {
-    return []
-  }
-}
+const getReferencingFiles = async (workspace: EditorWorkspace, id: ElemID): Promise<string[]> =>
+  awu(await workspace.getElementIncomingReferences(id.createBaseID().parent))
+    .concat(
+      id.idType === 'type'
+        ? awu(await (await workspace.elements).list()).filter(
+            elemId => elemId.idType === 'instance' && id.isEqual(new ElemID(elemId.adapter, elemId.typeName)),
```

Comment:
nit - I'd avoid creating a new element ID here, I don't think it is significantly better than comparing the parts separately
```suggestion
            elemId => elemId.adapter == id.adapter && elemId.typeName == id.typeName && elemId.idType === 'instance',
```